jax/numpy/lax_numpy.py

```diff
@@ -2371,48 +2371,35 @@ def outer(a, b, out=None):
     raise NotImplementedError("The 'out' argument to outer is not supported.")
   return ravel(a)[:, None] * ravel(b)
 
+@partial(jit, static_argnums=(2, 3, 4))
+def _cross(a, b, axisa, axisb, axisc):
+  a = moveaxis(a, _canonicalize_axis(axisa, ndim(a)), -1)
```

Comment:
You don’t really need `_canonicalize_axis` here anymore since moveaxis should do that internally.

Reply:
Done.